src/crate/crash/command.py

```diff
@@ -141,6 +143,8 @@ def _conf_or_default(key, value):
     parser.add_argument('-W', '--password', action='store_true',
                         dest='force_passwd_prompt', default=_conf_or_default('force_passwd_prompt', False),
                         help='force a password prompt')
+    parser.add_argument('-s', '--schema', type=str,
```

Comment:
Would it make sense to only allow the long option name? In case we want to use `-s` for something else, e.g. for pgsql compat or something.

Reply:
Good point!